src/tendencies_parametrizations.jl

```diff
@@ -1,23 +1,35 @@
 """
-Compute physical parametrization tendencies.
+    parametrization_tendencies!(diagn::DiagnosticVariables,
+                                M::PrimitiveEquationModel)
+
+Compute tendencies for u,v,temp,humid from physical parametrizations.
+Extract for each vertical atmospheric column the prognostic variables
+(stored in `diagn` as they are grid-point transformed), loop over all
+grid-points, compute all parametrizations on a single-column basis,
+then write the tendencies back into a horizontal field of tendencies.
 """
-function parametrization_tendencies!(
-    Prog::PrognosticVariables{NF},
-    Diag::DiagnosticVariables{NF},
-    M,
-) where {NF<:AbstractFloat}
-    @unpack pres_grid = Diag.grid_variables
+function parametrization_tendencies!(   diagn::DiagnosticVariables{NF},
+                                        M::PrimitiveEquationModel,
+                                        ) where NF
+
+    G = M.geometry
+    column = ColumnVariables{NF}(nlev=diagn.nlev)
```

Comment:
Will we need to make this thread-safe when it comes to parallelisation? I'm more familiar with multi-threading rather than multi-processing so not 100% sure how it'd work in the latter case.

Reply:
Yes, but that should be just making sure that every thread/processor has its own copy of column so that they don't interfere